test/dynamo/test_unspec.py

```diff
@@ -832,7 +832,7 @@ def forward(self, x: torch.Tensor, val: int) -> torch.Tensor:
         x1 = torch.rand(3, 5, 4, 8).to(memory_format=torch.channels_last)
         x2 = torch.rand(1, 5, 4, 8).to(memory_format=torch.channels_last)
 
-        torch._dynamo.decorators.mark_unbacked(x1, 0)
+        torch._dynamo.decorators.mark_unbacked(x1, 0, assume_contiguous=False)
```

Comment:
This is a kind of weird way to spell the API, because the contiguity of x1 is not really associated with a particular dimension, instead, it's the entire tensor!